src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -193,6 +200,17 @@ export const TransactionGenericRow = ({
                     IBC
                   </Tag>
                 )}
+                {isNft && (
+                  <Tag
+                    size='sm'
+                    colorScheme='blue'
+                    variant='subtle'
+                    minHeight={{ base: '1.2rem', md: compactMode ? '1.2rem' : '1.25rem' }}
+                    px={{ base: 2, md: compactMode ? 2 : 2 }}
+                  >
+                    NFT
```

Comment:
We can just be generic or always tag as `ERC-721` and `ERC-1155`.

@reallybeard you have a pref?